src/timeAgoInWords.test.ts

```diff
@@ -70,6 +92,112 @@ describe('timeAgoInWords', () => {
 		).toBe('Friday 15 Nov 2019');
 	});
 
+	it('does not pluralise the unit when the delta is one', () => {
+		const onHourAgo = new Date(Date.UTC(2019, 10, 17, 11, 0, 0)).getTime();
+		expect(
+			timeAgoInWords(onHourAgo, {
+				format: 'short',
+			}),
+		).toBe('1h');
+		expect(
+			timeAgoInWords(onHourAgo, {
+				format: 'med',
+			}),
+		).toBe('1h ago');
+		expect(
+			timeAgoInWords(onHourAgo, {
+				format: 'long',
+			}),
+		).toBe('1 hour ago');
+	});
+
+	it('only pluralises the unit when format is long', () => {
+		const threeHoursAgo = new Date(
+			Date.UTC(2019, 10, 17, 9, 0, 0),
+		).getTime();
+		expect(
+			timeAgoInWords(threeHoursAgo, {
+				format: 'short',
+			}),
+		).toBe('3h');
+		expect(
+			timeAgoInWords(threeHoursAgo, {
+				format: 'med',
+			}),
+		).toBe('3h ago');
+		expect(
+			timeAgoInWords(threeHoursAgo, {
+				format: 'long',
+			}),
+		).toBe('3 hours ago');
+	});
```

Comment:
the [`it.each` table format][1] could be useful here!

[1]: https://jestjs.io/docs/api#testeachtablename-fn-timeout

```ts
it([
  ['short', '3h'],
  ['med', '3h ago'],
  ['long', '3 hours ago'],
])('only pluralises the unit when format is long', (length, expected) => {
		const threeHoursAgo = new Date(
			Date.UTC(2019, 10, 17, 9, 0, 0),
		).getTime();
		expect(
			timeAgoInWords(threeHoursAgo, {
				format: length,
			}),
		).toBe(expected);
	});
```

Reply:
My feeling is with tests, you want them to be as boring and as simple as possible so that if they fail in future the developer who has that error message on his screen can immediately (hopefully) see what the problem is. We want to give our future selfs the easiest experience we can. I worry this pattern could confuse future developers - it certainly confused me! Sure, it's perhaps only 30 seconds effort to google it and work it out, but that's a lot when you're just trying to make the tests pass!

Reply:
Sure, I just suggest it because it has the added benefit of creating three _distinct_ tests, and you can identify exactly which combination of arguments fails.

Carry on!